ci-operator/step-registry/openshift-qe/installer/bm/day2/openshift-qe-installer-bm-day2-ref.yaml

```diff
@@ -0,0 +1,60 @@
+ref:
+  as: openshift-qe-installer-bm-day2
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-installer-bm-day2-commands.sh
+  credentials:
+  - namespace: test-credentials # this entry injects the custom credential
+    name: perfscale-metal-bastion
+    mount_path: /secret
```

Comment:
You need to switch this to `/bm` as you have defined here:  https://github.com/openshift/release/pull/52001/files#diff-0aee60055ce70da0627bc5c3219de2c2a868fd1bef0523d3249492e1c4e9b10aR9 Or switch the command reference to use `/secret`

Reply:
Thanks!